public/src/util/mediatypes.js

```diff
@@ -291,9 +290,6 @@ define( [ "localized", "util/uri", "util/xhr", "json!/api/butterconfig", "jquery
           if ( !respData ) {
             return errorCallback( CLYP_EMBED_UNPLAYABLE );
           }
-          if ( respData.Status !== "Public" ) {
```

Comment:
Looks good to me. As long as the Clyp API returns a 200, the audio file content should be considered importable.